core/src/main/java/hudson/PluginManager.java

```diff
@@ -1347,7 +1347,7 @@ public void stop() {
      */
     @Restricted(DoNotUse.class) // WebOnly
     public HttpResponse doPlugins() {
-        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        Jenkins.get().checkPermission(Jenkins.SYSTEM_READ);
```

Comment:
Pretty sure this is only used during the setup wizard, so doesn't need the permission check changed.

Reply:
```suggestion
        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```